README.md

```diff
@@ -45,73 +49,60 @@ haxelib install hxCodec 2.5.1
-
-## Credits menu
-![Screenshot_1](https://user-images.githubusercontent.com/44785097/144632635-f263fb22-b879-4d6b-96d6-865e9562b907.png)
-* You can add a head icon, name, description and a Redirect link for when the player presses Enter while the item is currently selected.
-
-## Awards/Achievements
-* The engine comes with 16 example achievements that you can mess with and learn how it works (Check Achievements.hx and search for "checkForAchievement" on PlayState.hx)
-
-## Options menu:
-* You can change Note colors, Delay and Combo Offset, Controls and Preferences there.
- * On Preferences you can toggle Downscroll, Middlescroll, Anti-Aliasing, Framerate, Low Quality, Note Splashes, Flashing Lights, etc.
-
-## Other gameplay features:
-* When the enemy hits a note, their strum note also glows.
-* Lag doesn't impact the camera movement and player icon scaling anymore.
-* Some stuff based on Week 7's changes has been put in (Background colors on Freeplay, Note splashes)
-* You can reset your Score on Freeplay/Story Mode by pressing Reset button.
-* You can listen to a song or adjust Scroll Speed/Damage taken/etc. on Freeplay by pressing Space.
+### Features
+
+## New title screen
+
+Make's the titlestate.hx a little bit better, including tweening and that sort of stuff
+
+![](https://user-images.githubusercontent.com/107285739/185762607-78af4bee-57f8-472b-908c-d2d7fb4e2d12.png)
+
+## Shaders VIA LUA/SOURCE
+* You can add: ```addWavyEffect('whateverthehellyourstageis', 1.5, 3);``` to stage .lua
+
+* You can also add it though source code, if you don't like sneaky code thefter's!
+
+**  
+```
+var bg:BGSprite = new BGSprite('folder/whateverthehellyourstageis', -600, -200, 0.9, 0.9);
+				add(bg);
+				bg.active = true;
+
+				var testshader:Shaders.GlitchEffect = new Shaders.GlitchEffect();
+				testshader.waveAmplitude = 0.1;
+				testshader.waveFrequency = 5;
+				testshader.waveSpeed = 2;
+
+				bg.shader = testshader.shader;
+				curbg = bg
+```
+## Better main menu
+
+Adds in some cool stuff
+
+![](https://user-images.githubusercontent.com/107285739/185762921-3fe34263-7341-4081-b39e-f005052879ee.png)
+
+## Camera follow + Gab-Engine UI
+
+* Adds in a hardcoded camera movement like in bambi's purgatory, and it is disablable whenever you feel like it!
+
+![](https://user-images.githubusercontent.com/107285739/185763095-231a84f3-deba-4be9-a072-631b9b88ff10.png)
+
+* Unique UI toggalble in the option's menu. This include's not limited to (not sure if you can disable color jsons but whatever):
+** Diffrent placement in accuracy, and a total note hit's! (judgement counter enabled as well, confusing with it but whatever)
+
+![](https://user-images.githubusercontent.com/107285739/185763122-d44cfaab-5e46-4945-8918-fd7da05cae10.png)
+
+## Freeplay sections
+
+* Make's it possible to play either mod's or the base game for FNF
+** Mods section can be enabled as soon you enable mods, in the mods menu state
```

Comment:
mods menu state is currently in the works, trying to fix this one thing where it makes weeks "tutorial" to week6 appear in the mods menu state